src/document.h

```diff
@@ -1279,14 +1279,27 @@ struct Document {
                     returnmessage = CopyImageToClipboard(c);
                 } else {
                     if (wxTheClipboard->Open()) {
-                        wxString s;
+                        wxString s, html;
                         if (k == A_COPYCT) {
                             loopallcellssel(c, true) if (c->text.t.Len()) s += c->text.t + " ";
                         } else {
                             s = selected.g->ConvertToText(selected, 0, A_EXPTEXT, this);
                         }
                         sys->clipboardcopy = s;
-                        wxTheClipboard->SetData(new wxTextDataObject(s));
+                        html = selected.g->ConvertToText(selected, 0, A_EXPHTMLT, this);
+                        wxDataObjectComposite *copyobj = new wxDataObjectComposite();
+                        auto *htmlobj = 
+                        // wxGTK crashes when the HTML Data Object is requested
+                        // so workaround this issue with a custom data object
+                        #ifdef __WXGTK__
+                            new wxCustomDataObject(wxDF_HTML);
+                        htmlobj->SetData(html.Len(), html);
+                        #else
+                            new wxHTMLDataObject(html);
+                        #endif
+                        copyobj->Add(new wxTextDataObject(s), false);
+                        copyobj->Add(htmlobj, true);
+                        wxTheClipboard->SetData(copyobj);

```

Comment:
How does this work?
If I paste in a text editor, am I going to get text or html?
It is important text copy/paste is preserved, so if the answer is html, then we need a separate Copy as HTML operation.